js/about/preferences.js

```diff
@@ -323,6 +323,83 @@ class BitcoinDashboard extends ImmutableComponent {
   }
 }
 
+class PaymentHistory extends ImmutableComponent {
+  get ledgerData () {
+    return this.props.ledgerData
+  }
+
+  render () {
+    const transactions = this.props.ledgerData.get('transactions')
+
+    return <div id='paymentHistory'>
+      <table className='sort'>
+        <thead>
+          <tr>
+            <th className='sort-header' data-l10n-id='date' />
+            <th className='sort-header' data-l10n-id='totalAmount' />
+            <th className='sort-header' data-l10n-id='receiptLink' />
+          </tr>
+        </thead>
+        <tbody>
+        {
+          transactions.map(function (row) {
+            return <PaymentHistoryRow transaction={row} ledgerData={this.props.ledgerData} />
+          }.bind(this))
+        }
+        </tbody>
+      </table>
+    </div>
+  }
+}
+
+class PaymentHistoryRow extends ImmutableComponent {
+
+  get transaction () {
+    return this.props.transaction
+  }
+
+  get timestamp () {
+    return this.transaction.get('submissionStamp')
+  }
+
+  get formattedDate () {
+    return formattedDateFromTimestamp(this.timestamp)
+  }
+
+  get numericDateStr () {
+    return (new Date(this.timestamp)).toLocaleDateString().replace(/\//g, '-')
+  }
+
+  get ledgerData () {
+    return this.props.ledgerData
+  }
+
+  get satoshis () {
+    return this.transaction.get('contribution').get('satoshis')
+  }
+
+  get currency () {
+    return this.transaction.get('contribution').get('fiat').get('currency')
+  }
+
+  get totalAmount () {
+    var fiatAmount = this.transaction.get('contribution').get('fiat').get('amount')
+    return fiatAmount.toFixed(2)
+  }
+
+  render () {
+    var date = this.formattedDate
+    var totalAmountStr = this.totalAmount + ' ' + this.currency
+    var receiptFileName = 'brave_ledger' + this.numericDateStr + '.pdf'
```

Comment:
this is minor, but we prefer to use JS template strings here. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals